### PR TITLE
Describe how to add an item to the gift register

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to add an item to the gift register

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/gift-registries.html

### CODE REVIEW

File: src/marketing/gift-registries.md [hunk 1]
```diff
@@ -23,3 +23,15 @@ Registry visitors who want to make a purchase can add the item to their carts di
 1. **Invitations are sent**. A link to the gift registry is included in each invitation. If [Gift Registry Search]({% link marketing/gift-registry-search.md %}) is available in the store, customers can search for specific gift registries by name, email address, or gift registry ID.
 
 1. **Invitation recipients place orders**. Those who receive an invitation can place an order for any item directly from the gift registry. As items are sold, Magento updates the gift registry item counts, and notifies the gift registry owner.
+
+## Add an item to the Gift Register
```
Comment (leslietilling) [inline src/marketing/gift-registries.md:27]:
```suggestion
## Add an item to a gift registry on the storefront
```

File: src/marketing/gift-registries.md [hunk 2]
```diff
@@ -23,3 +23,15 @@ Registry visitors who want to make a purchase can add the item to their carts di
...
+
+1. Open the product that you want to add to the gift registry event.
```
Comment (leslietilling) [inline src/marketing/gift-registries.md:29]:
```suggestion
1. The customer opens the product that they want to add to the gift registry event.
```

File: src/marketing/gift-registries.md [hunk 3]
```diff
@@ -23,3 +23,15 @@ Registry visitors who want to make a purchase can add the item to their carts di
...
+
+1. Click <span class="btn">Add to Cart</span>.
```
Comment (leslietilling) [inline src/marketing/gift-registries.md:31]:
```suggestion
1. Clicks/taps <span class="btn">Add to Cart</span>.
```

File: src/marketing/gift-registries.md [hunk 4]
```diff
@@ -23,3 +23,15 @@ Registry visitors who want to make a purchase can add the item to their carts di
...
+
+1. Click <span class="btn">View and Edit Cart</span> on the Mini Cart.
```
Comment (leslietilling) [inline src/marketing/gift-registries.md:33]:
```suggestion
1. Clicks/taps <span class="btn">View and Edit Cart</span> on the mini cart.
```

File: src/marketing/gift-registries.md [hunk 5]
```diff
@@ -23,3 +23,15 @@ Registry visitors who want to make a purchase can add the item to their carts di
...
+
+1. On the Shopping Cart page select the event you need and click <span class="btn">Add All To Gift Registry</span>.
```
Comment (leslietilling) [inline src/marketing/gift-registries.md:35]:
```suggestion
1. On the Shopping Cart page, selects the event they want and clicks/taps <span class="btn">Add All To Gift Registry</span>.
```

File: src/marketing/gift-registries.md [hunk 6]
```diff
@@ -23,3 +23,15 @@ Registry visitors who want to make a purchase can add the item to their carts di
...
+
+1. Items will be added to the gift register of the selected event.
```
Comment (leslietilling) [inline src/marketing/gift-registries.md:37]: This is a result and not a step

```suggestion
   Items are added to the gift registry of the selected event.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
If we document a workflow on the storefront, we are describing the required action of the customer to the store merchant. So in these cases, it is a third-person procedure. We have not been consistent with this in the guide but it is something that we want to address going forward.

File: src/marketing/gift-registries.md (hunk 6 above)
Comment (ZhovnirMaria) [reply src/marketing/gift-registries.md]: Done, thanks 😊

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**